src/index.html

```diff
@@ -5,8 +5,392 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__image">
+        <img src="./images/logo.png" alt="MOYOlogo">
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="catalog">
+        <article class="card catalog__card"  data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="Apple iMac"
+            class="product-photo"
+          >
+
+          <div class="product">
+            <div class="product__info">
+              <h3 class="product__title">
+                Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)
+              </h3>
+              <p>Код товара: 195434</p>
+            </div>
+
+            <div class="product__rating">
+              <div class="product__stars stars">
```

Comment:
add modifier, so that changing modifier would change number of active stars accordingly (and styles for all the modifiers: `stars--1, stars--2`, etc.)